views/update/update-review-overseas-entity-information.html

```diff
@@ -0,0 +1,29 @@
+{% extends "layout.html" %}
+
+{% set title = "You're about to review overseas entity information" %}
+
+{% block pageTitle %}
+  {% include "includes/page-title.html" %}
+{% endblock %}
+
+{% block backLink %}
+  {% include "includes/back-link.html" %}
+{% endblock %}
+
+{% block content %}
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <span class="govuk-caption-xl govuk-!-padding-bottom-1">{{ appData.entity_name }} - {{ appData.entity_number }}</span>
+    <h1 class="govuk-heading-xl">{{ title }}</h1>
+    <form method="post">
+      <p class="govuk-body">We'll show you the information we hold about the overseas entity</p>
+      <p class="govuk-body">You'll need to:</p>
+      <ul class="govuk-list govuk-list--bullet">
+        <li>check the information is correct, and update it if needed</li>
+        <li>enter the overseas entity's email address</li>
+      </ul>
```

Comment:
I think all this content block from here needs re-worked to use something from the govuk library instead of writing out each HTML element from line 19-24

Reply:
For example, I think your page if you were to display it locally doesn't have a h1 heading which wouldn't pass for accessibility checks etc.

Reply:
I can't see any different approaches being used to where bullet pointed lists are required elsewhere